browser/search_engines/search_engine_provider_service.cc

```diff
@@ -39,25 +39,9 @@ SearchEngineProviderService::SearchEngineProviderService(
       base::BindRepeating(&SearchEngineProviderService::OnPreferenceChanged,
                           base::Unretained(this)));
 
-  std::vector<TemplateURLPrepopulateData::BravePrepopulatedEngineID>
```

Comment:
@bsclifton  Should I leave this logic as-is? this alternative url is used for private/guest profile.

Reply:
I think this is OK 😄 Like you shared, this is only used in Tor window and then private window when "use DDG" flag is enabled